packages/scripts/scripts/test-e2e.js

```diff
@@ -50,4 +50,16 @@ if ( hasCliArg( '--puppeteer-interactive' ) ) {
 	process.env.PUPPETEER_SLOWMO = getCliArg( '--puppeteer-slowmo' ) || 80;
 }
 
+const configsMapping = {
+	WP_BASE_URL: '--wordpress-host',
```

Comment:
Do you think we should align with the env variable name? `--wordpress-base-url` maybe. I forgot that WordPress can be also installed in subfolder and seeing the actual implementation made me think :)

Reply:
Yes, that's better, since there is no need to think twice why the argument name is describing a different thing than expected from env key.